package.json

```diff
@@ -101,14 +103,14 @@
     "terra-disclosure-manager": "^4.9.0",
     "url-loader": "^4.1.1",
     "uuid": "^3.0.0",
-    "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.3.1",
+    "webpack": "^5.28.0",
```

Comment:
I'm not sure if it's OK to bump webpack, the cli and the dev server to new major versions as a part of this project. @JessieRandle FYI, if not we might need to rethink scope?

Reply:
@xenoworf will try to downgrade it, but before Ben Cai asked us update webpack to 5 versio